content/en/dashboards/ddsql_editor/reference/_index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: DDSQL Reference
+---
+
+{{< callout url="https://google.com">}}
+DDSQL is in private beta.
+{{< /callout >}}
+
+## Overview
+
+DDSQL is a query language for Datadog data. It implements several standard SQL functions, such as `SELECT`, along with new functions for working with Datadog tags, metrics, and so on. For example, [the `AGGR` statement][5], which aggregates metrics data, is unique to DDSQL.
+
+You can use DDSQL to query your infrastructure in the [DDSQL Editor][2] in Datadog.
```

Comment:
as scope is outside just infrastructure (RUM, Metrics) - might be worth rephrasing.